core/templates/services/local-storage.service.spec.ts

```diff
@@ -204,5 +204,32 @@ describe('LocalStorageService', () => {
 
       expect(callbackFnSpy).toHaveBeenCalled();
     });
+
+    it('should correctly save unique progress url id', () => {
```

Comment:
Ditto here and below: url -> URL, id -> ID

Reply:
Done!